README.md

```diff
@@ -73,31 +73,44 @@ Note that unlike `JSON.stringify()`, object property ordering is not considered.
 
 A re-export of `faker` from [`@faker-js/faker`](https://github.com/faker-js/faker). We do not alter faker in any way, and do not seed it.
 
-### `copycat.oneOf(input, values)`
+### `copycat.scramble(string[, options])`
 
-Takes in an [`input`](#input) value and an array of `values`, and returns an item in `values` that corresponds to that `input`:
+Takes in a `string` value, and returns a string with the same length, but with each character replaced with a different character in the same character range:
+* By default, spaces are preserved (see `preserve` option below)
+* Lower case ascii characters are replaced with lower case ascii letters
+* Upper case ascii characters are replaced with upper case ascii letters
+* Digits are replaced with digits
+* Any other ascii character in the code point range `32` to `126` (`0x20 - 0x7e`) is replaced with an ascii in the same range
+* Any other character is replaced with a [Latin-1](https://en.wikipedia.org/wiki/Latin-1_Supplement) character in the range of (`0x20 - 0x7e`, or `0xa0 - 0xff`)
```

Comment:
This behaviour was want seemed most intuitive to me, but happy to discuss alternatives

Reply:
Like for like scrambling makes sense, as it will preserve the aesthetics of the data.